.circleci/config.yml

```diff
@@ -687,7 +687,7 @@ jobs:
           name: move compiled ui files to agent/uiserver
           command: |
             rm -rf agent/uiserver/dist
-            mv ui/packages/consul-ui/dist agent/uiserver
+            mv packages/consul-ui/dist agent/uiserver
```

Comment:
The glob `packages/consul-ui/dist` is persisted to workspace root, not `ui/`. So once we use `attach_workspace.at: .` the directory will actually be at the root of the workspace